meeting-notes/2019/Q2/2019-06-03--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,213 @@
+  - integrate `interface-connection` proposal
+    - [libp2p/js-libp2p-tcp#102](https://github.com/libp2p/js-libp2p-tcp/pull/102)
+    - [libp2p/js-libp2p-websockets#82](https://github.com/libp2p/js-libp2p-websockets/pull/82)
+ - more async iterators work
+ 
+@vmx
+ - Done:
+   - https://github.com/vmx/ipld-graphviz Supports [Parasol](http://parasol.alexgirard.com/) JSON output now
+   - multiformats cleanup:
+     - I try to archive multistream: https://github.com/multiformats/multistream/issues/9
+     - Try to make multicodec README clearer: https://github.com/multiformats/multicodec/pull/134
+ - Blocked: Nope
+ - Next:
+   - More work on [STAC](https://stacspec.org/) on IPLD
+   - Playing a bit with WASM in regards to IPLD
+
+@jimpick
+ - Done:
+   - More IPFS Camp Course C - "Apps"
+   - Got HTTP Signed Exchanges "publisher" demo working (SXG Demo):
+     https://github.com/ipfs/integration-mini-projects/issues/3
+ - Blocked:
+   - Need to figure out ipfs-cluster issues for peer-base-pinner
+ - Next:
+   - More IPFS Camp
+   - Switch over to Testlab - got some ideas for content discovery tests
+   - investigate some possible bugs revealed by SXG demo
+
+
+@hugomrdias
+ - Done:
+   - https://github.com/ipfs/js-ipfs-unixfs-importer/pull/31 feat: use a rabin chunker in wasm
+   - https://github.com/ipfs/js-ipfs/pull/2110 Prepare for new aegir version
+   - https://github.com/ipfs/js-ipfs-http-client/pull/1021 Prepare for new aegir version
+   - IPFS Camps Sessions
+   - Dns resolver for IPNS
+ - Blocked:
+ - Next:
+   - Continue DNS resolver
+   - IPFS Camp Sessions
+
+@achingbrain
+ - Done:
+   - async/await for js-datastore-{fs|level|core}
+   - async/await for js-ipfs-repo
+   - Moved house
+ - Blocked:
+   - N/a
+ - Next:
+   - merge Dirk's mortice PR
+   - merge Hugo's unixfs-importer rabin WASM PR
+   - libp2p workshop for IPFS Camp
+   - npm-in-a-box for IPFS Camp Science Fair
+   - Upgrade npm-on-ipfs deps
+   - Deploy npm-on-ipfs somewhere faster
+
+@dirkmc
+ - Done:
+   - Fixed some bugs with DHT, bitswap
+     - [DHT: send correct payload in ADD_PROVIDER RPC](https://github.com/libp2p/js-libp2p-kad-dht/pull/127)
+     - [DHT: favour providers peerInfo over sender peerInfo in ADD_PROVIDER](https://github.com/libp2p/js-libp2p-kad-dht/pull/129)
+     - [Bitswap: ignore unwanted blocks](https://github.com/ipfs/js-ipfs-bitswap/pull/194)
+   - Completed [GC implementation](https://github.com/ipfs/js-ipfs/pull/2022) (mark and sweep)
+   - Created [RFC Issue](https://github.com/ipfs/js-ipfs/issues/2148) for js-ipfs profiles
+   - Reviewing [repo migration tool](https://github.com/AuHau/js-ipfs-repo-migrations/pull/1) being created by a community member
+ - Blocked:
+   - GC PR depends on a [PR against mortice](https://github.com/achingbrain/mortice/pull/1)
+ - Next:
+   - Implement profiles
+
+@name
+ - Done:
+ - Blocked:
+ - Next:
+
+### Questions
+
+- When should we target DHT being ready for real-world use? (dirkmc question)
```

Comment:
@dirkmc The original plan was to get this released before IPFS Camp. I've updated the PR to enable it by default to include the dependencies that need to be resolved before it is mergable: https://github.com/ipfs/js-ipfs/pull/1994

My bad - this should have already been documented.